api/src/main/java/org/openmrs/module/muzimafingerPrint/PatientJsonParser.java

```diff
@@ -4,12 +4,23 @@
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
-import org.openmrs.*;
+import org.openmrs.Patient;
+import org.openmrs.PersonName;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Location;
+import org.openmrs.User;
 import org.openmrs.api.context.Context;
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.*;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.ArrayList;
 
 /**
  * Created by vikas on 17/02/15.
```

Comment:
We want to remove this kind of comments which are coming from the template from intellij or eclipse :)

Please remove this later on.